test/tests.py

```diff
@@ -65,8 +65,7 @@ def active_test(fullname):
 bat_sfx = batch_suffix()
 exe_sfx = '.exe' if is_windows() else ""
 
-mini_python = [join(simics_base_path(), host_type(), "bin",
-                    "mini-python" + exe_sfx)]
+mini_python = [join(project_path(), "bin", f"mini-python{batch_suffix()}")]
```

Comment:
Actually, either use `testparams.project_path`, or update existing usages of  `testparams.project_path` to use `project_path`